HIRS_ProvisionerTPM2/include/CommandTpm2.h

```diff
@@ -86,8 +86,9 @@ class CommandTpm2 {
 
     std::string getPublicArea(const std::string& filename);
 
-    std::string runTpm2CommandWithRetry(const std::string& command,
-                                        const std::string& args);
+    std::string runTpm2CommandWithRetry(const std::string & command,
```

Comment:
Can we axe the spaces between the ampersand and the type of the parameter? We landed on that convention quite a while ago.